packages/components/psammead-media-indicator/src/index.stories.jsx

```diff
@@ -52,6 +52,20 @@ storiesOf('Components|MediaIndicator/Video', module)
       />
     ),
     { notes },
+  )
+  .add(
+    'inline media indicator',
+    ({ service }) => (
+      <>
+        <MediaIndicator
+          type="video"
+          service={service}
+          isInline={boolean('inline?', true)}
+        />
+        {text('extra text', 'example text')}
```

Comment:
Now that we have `children` as a prop, we should pass in the example text as a child node. 
e.g. `<MediaIndicator type="video"....>{text('extra text', 'example text')}</MediaIndicator>`